troposphere/static/js/components/modals/volume/VolumeAttachModal.react.js

```diff
@@ -227,12 +228,14 @@ define(
             instances = this.state.instances,
             content;
 
-        if (!instances) {
+        var activeInstances = instances && instances.filter(i => i.is_active());
+
+        if (instances == null) {
           content = this.renderLoadingContent();
-        } else if (instances.length <= 0) {
-          content = this.renderAttachRulesContent();
+        } else if (activeInstances.length > 0) {
+          content = this.renderAttachVolumeContent(activeInstances);
         } else {
-          content = this.renderAttachVolumeContent(instances);
+          content = this.renderAttachRulesContent(); 
         }
 
         return (
```

Comment:
Steve has asked me what is up with all the *red* here. I think it's the unmatched `'`. I don't know if we should add an HTML entity, `&apos;`, there or put this instead inside a template string.